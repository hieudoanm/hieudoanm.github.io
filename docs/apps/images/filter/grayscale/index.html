<!DOCTYPE html><html lang="en"><head><meta charSet="utf-8" data-next-head=""/><meta name="viewport" content="width=device-width" data-next-head=""/><title data-next-head="">Nothing</title><link rel="icon" type="image/x-icon" href="/favicon.ico"/><link rel="manifest" href="/manifest.json"/><meta name="theme-color" content="#f3f4f6"/><link rel="preload" href="/_next/static/media/93f479601ee12b01-s.p.woff2" as="font" type="font/woff2" crossorigin="anonymous" data-next-font="size-adjust"/><link rel="preload" href="/_next/static/css/2410e685df8c83a7.css" as="style"/><link rel="stylesheet" href="/_next/static/css/2410e685df8c83a7.css" data-n-g=""/><noscript data-n-css=""></noscript><script defer="" noModule="" src="/_next/static/chunks/polyfills-42372ed130431b0a.js"></script><script src="/_next/static/chunks/webpack-3804331ed1fd6dff.js" defer=""></script><script src="/_next/static/chunks/framework-b8820ca12ff52ecf.js" defer=""></script><script src="/_next/static/chunks/main-cd72e0cac4836575.js" defer=""></script><script src="/_next/static/chunks/pages/_app-c887ab517bc29b4a.js" defer=""></script><script src="/_next/static/chunks/4fd45486-e8676c06c5ccf2a2.js" defer=""></script><script src="/_next/static/chunks/pages/apps/images/filter/grayscale-5694b53c26efc988.js" defer=""></script><script src="/_next/static/2b9gWQH1ngIXmi7nB1DJy/_buildManifest.js" defer=""></script><script src="/_next/static/2b9gWQH1ngIXmi7nB1DJy/_ssgManifest.js" defer=""></script></head><body class="bg-gray-100 text-gray-900 antialiased"><div id="__next"><div class="__className_02660c"><div class="container mx-auto p-8"><div class="flex flex-col gap-y-4"><label for="upload-image" class="cursor-pointer border border-dotted px-4 py-2 text-center"><input type="file" accept="image/*" id="upload-image" class="hidden" name="image"/><span>Upload File</span></label><button class="w-full" style="aspect-ratio:3 / 2"><div class="h-full w-full"><div class="h-full w-full bg-cover bg-center" style="background-image:url(/_next/static/media/film.d64a9c6f.jpg)"></div></div></button><button type="button" class="w-full cursor-pointer bg-gray-900 px-4 py-2 text-gray-100">Download</button></div></div></div></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{}},"page":"/apps/images/filter/grayscale","query":{},"buildId":"2b9gWQH1ngIXmi7nB1DJy","nextExport":true,"autoExport":true,"isFallback":false,"scriptLoader":[]}</script></body></html>