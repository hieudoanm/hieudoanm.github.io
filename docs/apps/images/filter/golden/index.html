<!DOCTYPE html><html lang="en"><head><meta charSet="utf-8" data-next-head=""/><meta name="viewport" content="width=device-width" data-next-head=""/><title data-next-head="">HIEU</title><link rel="icon" type="image/x-icon" href="/nothing/favicon.ico" data-next-head=""/><link rel="preload" href="/_next/static/media/93f479601ee12b01-s.p.woff2" as="font" type="font/woff2" crossorigin="anonymous" data-next-font="size-adjust"/><link rel="preload" href="/_next/static/css/f3992dae9c855df7.css" as="style"/><link rel="stylesheet" href="/_next/static/css/f3992dae9c855df7.css" data-n-g=""/><noscript data-n-css=""></noscript><script defer="" noModule="" src="/_next/static/chunks/polyfills-42372ed130431b0a.js"></script><script src="/_next/static/chunks/webpack-ca25dbde007283fc.js" defer=""></script><script src="/_next/static/chunks/framework-da1c48dc673ca549.js" defer=""></script><script src="/_next/static/chunks/main-8aad74cc669db8f5.js" defer=""></script><script src="/_next/static/chunks/pages/_app-41b942832c5ff374.js" defer=""></script><script src="/_next/static/chunks/bf9d8ce5-9f7306c40178035c.js" defer=""></script><script src="/_next/static/chunks/pages/apps/images/filter/golden-3a80d8908e73f868.js" defer=""></script><script src="/_next/static/pxDamZdV54OB6_sCDSDdT/_buildManifest.js" defer=""></script><script src="/_next/static/pxDamZdV54OB6_sCDSDdT/_ssgManifest.js" defer=""></script></head><body class="antialiased"><div id="__next"><div class="__className_b3860d"><div class="container mx-auto p-8"><div class="flex flex-col gap-y-4"><label for="upload-image" class="cursor-pointer border border-dotted px-4 py-2 text-center"><input type="file" accept="image/*" id="upload-image" class="hidden" name="image"/><span>Upload File</span></label><button class="w-full" style="aspect-ratio:3 / 2"><div class="h-full w-full"><div class="h-full w-full bg-cover bg-center" style="background-image:url(/_next/static/media/film.d64a9c6f.jpg)"></div></div></button><button type="button" class="w-full cursor-pointer bg-gray-900 px-4 py-2 text-gray-100">Download</button></div></div></div></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{}},"page":"/apps/images/filter/golden","query":{},"buildId":"pxDamZdV54OB6_sCDSDdT","nextExport":true,"autoExport":true,"isFallback":false,"scriptLoader":[]}</script></body></html>